.github/workflows/license-lint.yaml

```diff
@@ -0,0 +1,26 @@
+name: license-lint
+on: [push, pull_request]
+
+jobs:
+  license-lint:
+    name: "license-lint"
+    runs-on: ubuntu-22.04
```

Comment:
we have been using `ubuntu-latest` on other actions. do we need this specific version? i am okay to pin as long as we align on all of them.

Reply:
My preference would be for us to pin every version. While it can be work when we have to update, I think it can help reduce variance across all GH actions runs.

I can take on putting out a small PR that pins runner versions across the wfs.

Reply:
can you create an issue for this for tracking

Reply:
Thanks for following up here! I added one https://github.com/open-policy-agent/gatekeeper/issues/2527